internal/endtoend/testdata/func_match_types/sqlite/go/query.sql.go

```diff
@@ -0,0 +1,43 @@
+// Code generated by sqlc. DO NOT EDIT.
+// source: query.sql
+
+package querytest
+
+import (
+	"context"
+)
+
+const authorPages = `-- name: AuthorPages :many
+select author, count(title) as num_books, sum(pages) as total_pages
+from books
+group by author
+`
+
+type AuthorPagesRow struct {
+	Author     string
+	NumBooks   int64
+	TotalPages float64
```

Comment:
This should be an int64 once https://github.com/kyleconroy/sqlc/issues/1450 is fixed